code/modules/medical/genetics/bioEffects/harmful.dm

```diff
@@ -662,7 +662,16 @@
 		if (!istype(L) || (L.stat == 2))
 			return
 		if (probmult(prob_sting))
-			boutput(src, "<span class='alert'>A bee in your cloud stung you! How rude!</span>")
+			if (ishuman(L))
+				var/mob/living/carbon/human/H = L
+				if (prob(50))
+					if (istype(H.wear_suit, /obj/item/clothing/suit/bio_suit/beekeeper))
```

Comment:
Just to make sure, the intent here is a 50/50 whether the bees sting your head or your body?

Reply:
Yeah. I couldn't decide on a better split.